app/src/main/java/com/example/dogbook/ComposeActivity/ComposeActivity.java

```diff
@@ -143,6 +175,12 @@ private void uploadPost() {
         if (photoFile != null) {
             post.setPhoto(new ParseFile(photoFile));
         }
+        if (currentLocation != null) {
+            ParseGeoPoint parseGeoPoint = new ParseGeoPoint();
+            parseGeoPoint.setLatitude(currentLocation.getLatitude());
+            parseGeoPoint.setLongitude(currentLocation.getLongitude());
+            post.setLocation(parseGeoPoint);
+        }
```

Comment:
Hm, maybe we should pull this into a separate method -- "addLocationData()", but this is fine for now.  Would be worth considering if this grows even more